sdks/python/apache_beam/runners/portability/local_job_service.py

```diff
@@ -274,8 +274,9 @@ def _run_job(self):
         result = self._invoke_runner()
         _LOGGER.info(
             'Successfully completed job in %s seconds.', time.time() - start)
-        self.set_state(beam_job_api_pb2.JobState.DONE)
         self.result = result
+        result.wait_until_finish()
```

Comment:
Should this come before the Logger call since presumably the job hasn't finished yet?